HISTORY.rst

```diff
@@ -2,21 +2,21 @@
 History
 =======
 
-0.28.0 (unreleased)
+0.28.0 (2021-07-06)
 -------------------
 
 New features and enhancements
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 * Automatic load of translations on import and possibility to pass translations for virtual modules.
-* New ``xclim.testing.list_datasets`` function listing all available test datasets in repo `xclim-testdata`.
-* `spatial_analogs` accepts multi-indexes as the `dist_dim` parameter and will work with candidates and target arrays of different lengths.
-* `humidex` can be computed using relative humidity instead of dewpoint temperature.
+* New ``xclim.testing.list_datasets`` function listing all available test datasets in repo ``xclim-testdata``.
```

Comment:
I think we need some guidelines on how best to use the tick-formatting. Should we opt for always doubled (``) for call signatures, files, functions, etc.?

Reply:
That's my understanding. single ticks are for links. But happy to be proven wrong.